ci/azure-pipelines.yml

```diff
@@ -33,6 +33,8 @@ stages:
             py37:
             py38:
               coverage: true
+            py39:
+            py310:
```

Comment:
You need to update the templates version to `refs/tags/4.0` I think. I only added py39 and py310 support recently :) Maybe you'll need to add `pythonVersion` parameter to the `publish` step as well, see here: https://github.com/florimondmanca/azure-pipelines-templates/tree/master#job--python-publishyml